docs/hardware/spi-receivers.md

```diff
@@ -33,7 +33,9 @@ If your Flight Controller model is not in the list above, consult your Flight Co
     
     Likewise, the [v2.0 BetaFPV F4 1S 5A](https://betafpv.com/collections/brushless-flight-controller/products/f4-1s-5a-aio-brushless-flight-controller-elrs-2-4g) comes with an on-board UART-based ExpressLRS Receiver and doesn't use the SPI ExpressLRS implementation. Use the `BETAFPV AIO 2400 RX` Device target.
 
-Once flashed, you will need to paste in the `diff all` you have saved. Don't forget to type in `save` and press enter once done. Power cycle your flight controller, and you should be set. Review your Betaflight settings (no changes needed for the Receiver Type and Protocol; should already be set with `SPI Receiver`, with Provider as `ExpressLRS`).
+If you're on windows and flashing fails to start, you may need to correct the drivers using a tool like ImpulseRC Driver Fixer (link can be found on Betaflight Configurator's main page). 
```

Comment:
Please link the fixer, and include Zadig as another option.
Then mention they are linked in the Welcome Tab as well. If possible, put an image of these links in the Welcome Tab.

Capitalize "Windows". Or use "If you're in a Windows machine..."

Reply:
I'll add the ImpulseRC link, but I'm not keen on adding a link to Zadig. Mentioning it should be fine, but I'd like to steer users towards IRC Driver Fixer as newbies tend to do more damage than good when using Zadig. 

Will figure out how to add an image.

Reply:
Edited, added the screenshot, but I'm not going to add Zadig here. People who understand the issue well enough to use Zadig (without screwing up their VCP drivers) don't need this guide. 

Rebased to bring this back up to date.